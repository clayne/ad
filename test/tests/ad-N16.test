ad | -N16 | pjl-conductor-200.jpg | 0
