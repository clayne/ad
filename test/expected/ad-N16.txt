0000000000000000: FFD8 FFE1 020A 4578 6966 0000 4D4D 002A  ......Exif..MM.*
